etcdctl/READMEv3.md

```diff
@@ -484,6 +484,10 @@ On success, prints a humanized table of each endpoint URL, ID, version, database
 
 On success, prints a line of JSON encoding each endpoint URL, ID, version, database size, leadership status, raft term, and raft status. On failure, returns with a non-zero exit code.
 
+##### Protobuf reply
```

Comment:
we should document which command supports protobuf and links to the protobuf definition.